server/src/resolvers/roomsResolver.ts

```diff
@@ -1,125 +1,123 @@
-import {EquipmentRepository} from "../repositories/Equipment/EquipmentRepository";
-import {UserRepository} from "../repositories/Users/UserRepository";
-import {RoomRepo} from "../repositories/Rooms/RoomRepository";
+import { EquipmentRepository } from "../repositories/Equipment/EquipmentRepository";
+import { UserRepository } from "../repositories/Users/UserRepository";
+import { RoomRepo } from "../repositories/Rooms/RoomRepository";
 
 const er = new EquipmentRepository();
 const ur = new UserRepository();
 const rr = new RoomRepo();
 
 const RoomResolvers = {
-    Query: {
-        rooms: async (_: any, args: any, context: any) => {
-            try {
-                return await rr.getRooms();
-            } catch (e) {
-                console.log("Error:", e);
-            }
-        },
-        room: async (_: any, args: any, context: any) => {
-            try {
-                return await rr.getRoomByID(args.id);
-            } catch (e) {
-                console.log("Error:", e);
-            }
-        },
-        roomByEquipment: async (_: any, args: any, context: any) => {
-            try {
-                const equip = await er.getEquipmentById(args.equipmentID)
-                if (equip) {
-                    return await rr.getRoomByID(equip.roomID)
-                }
-            } catch (e) {
-                console.log("Error:", e);
-            }
-        },
-        roomByLabbie: async (_: any, args: any, context: any) => {
-            try {
-                const labbie = await ur.getUserByID(args.labbieID)
-                if (labbie) {
-                    return await rr.getRoomByID(labbie.roomID)
-                }
-            } catch (e) {
-                console.log("Error:", e);
-            }
-        },
+  Query: {
+    rooms: async (_: any, args: any, context: any) => {
+      try {
+        return await rr.getRooms();
+      } catch (e) {
+        console.log("Error:", e);
+      }
+    },
+    room: async (_: any, args: any, context: any) => {
+      try {
```

Comment:
These try catches might stop the Apollo client from knowing that any error happened which could be bad (like receiving null for rooms)

Reply:
Yup, agreed. I only touched this file to fix what I needed and was trying to keep this PR from growing out of control so I didn't refactor other stuff. Rest of the changes were Prettier reformatting.